### DISCUSSION TITLE
Add albums or stack of tracks to playlist

### DISCUSSION THREAD

**Discussion by maste9** [music-assistant/support#1767] [Feature requests and ideas]
When I yet wanted to summarize two albums into one playlist I had to notice that there's no way to that except of choosing one track at a time and add it to playlist.

 * It would be nice, if there was an __Add to playlist__ button in the play button's context menu of an album.

 * If I multichoose several tracks of a list, press __Action__ regarding those and choose __Add to playlist__ I would expect to find each track on that playlist after it. But actually I received a lot of errors like this in log:
```
mass             | 2024-01-05 17:37:55.484 WARNING (MainThread) [music_assistant.providers.filesystem_local] Could not parse uri/file filesystem_local2://track/02/fc/ec/13-hellblade_ost_hela_mp3.mp3p3 to track: track://02/fc/ec/13-hellblade_ost_hela_mp3.mp3p3 not found on provider filesystem_local2
```
and not more than 1 track found its way into the playlist.  (Maybe that's also kind of a bug report :grimacing: )

 * To transfer everything from the waiting list to a playlist would also be a neat feature.

Besides such special wishes: MA feels already quite complete as far as my needs are concerned. Impressive! :slightly_smiling_face:

**Comment by OzGav** [comment]
Yes the first point is valid. I will add a task to our list

on your second point that is already an open issue. Hopefully will be fixed soon https://github.com/music-assistant/hass-music-assistant/issues/1456

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
